cells/tullia/task.nix

```diff
@@ -112,10 +112,18 @@ in {
 
   nix-build = {config, ...}: {
     command.text = "nix build";
-    memory = 2 * 1024;
+    memory = 4 * 1024;
     preset.nix.enable = true;
+    env = {
+      NIX_CONFIG = ''
+        experimental-features = ca-derivations flakes nix-command recursive-nix
+        log-lines = 1000
+        show-trace = true
+        sandbox = false
+      '';
+    };
```

Comment:
Why? The nix preset already [does this](https://github.com/input-output-hk/tullia/blob/575362c2244498e8d2c97f72861510fa72e75d44/nix/preset/nix.nix#L24-L27).

Reply:
you can reproduce this issue with `tullia run doc`.
Looks like the nix_conf_dir does not in a right place.

Reply:
You're right! This only worked on container runtimes, but not nsjail. And it's better the way you did it in the new changes :+1: